test/plugin/scenarios/dbcp-2.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,240 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
+- serviceName: dbcp-2.x-scenario
+  segmentSize: nq 0
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: Mysql/JDBI/Connection/close
+      operationId: eq 0
+      parentSpanId: 1
+      spanId: 2
+      componentId: 33
+      isError: false
+      spanType: Exit
+      peer: mysql-server:3306
+      skipAnalysis: false
+      tags:
+        - {key: db.type, value: sql}
+        - {key: db.instance, value: test}
+        - {key: db.statement, value: ''}
+    - operationName: DBCP/Connection/getConnection
+      operationId: 0
+      parentSpanId: 0
+      spanId: 1
+      startTime: gt 0
+      endTime: gt 0
+      componentId: 102
+      isError: false
+      spanType: Local
+      peer: ''
+      skipAnalysis: false
+    - operationName: Mysql/JDBI/Statement/execute
+      operationId: 0
+      parentSpanId: 0
+      spanId: 3
+      startTime: gt 0
+      endTime: gt 0
+      componentId: 33
+      isError: false
+      spanType: Exit
+      peer: mysql-server:3306
+      skipAnalysis: false
+      tags:
+        - {key: db.type, value: sql}
+        - {key: db.instance, value: test}
+        - key: db.statement
+          value: "CREATE TABLE test_DBCP(\nid VARCHAR(1) PRIMARY KEY, \nvalue VARCHAR(1)\
+          \ NOT NULL)"
+    - operationName: DBCP/Connection/close
```

Comment:
@Jargon96 @dmsolr @ascrutae Does DBCP pool have a name? I hope this could be `DBCP/{pool-name}/...`. Then some of the operations like `get` and `close` should be the logic span later to show the connection pool performance.

Reply:
> @Jargon96 @dmsolr @ascrutae Does DBCP pool have a name? I hope this could be `DBCP/{pool-name}/...`. Then some of the operations like `get` and `close` should be the logic span later to show the connection pool performance.

@wu-sheng Could I use this ?  `DBCP/Apache-Commons-DBCP/...`

Reply:
See my comment below.

Reply:
@wu-sheng  said whether the pool has something like a name or others to identify it.
As far as I see, it does not have.

Reply:
I checked the API and found that DBCP doesn't have attribute `name` to set pool name.

Reply:
@dmsolr @ascrutae Thank you for checking. I didn’t find it either.

Reply:
OK, then let's keep the name as what it is today.